src/pvt.c

```diff
@@ -271,8 +271,9 @@ static u8 filter_solution(gnss_solution* soln, dops_t* dops)
    * regulations. It must not be removed. Any modification to this condition
    * is strictly not approved by Swift Navigation Inc. */
 
-  if (vector_norm(3, soln->vel_ecef) >= 0.514444444*1000)
-    /* Velocity is greater than 1000kts. */
+  if ((vector_norm(3, soln->vel_ecef) >= 0.514444444*1000)
+      && (soln->pos_llh[2] > 18000))
+    /* COCOM limits: velocity greater than 1000 kts and altitude > 18 km. */
```

Comment:
Hmm, per ECCN 7A105 there is indeed a pure velocity limit but it's actually 600 m/s not 1000 knots.  The 1000 knots / 18 km language has been deleted from the ITAR algoteher.  Anyway it's not related to this PR, I'll pull it out.